puppet/modules/kubernetes/templates/kube-apiserver.service.erb

```diff
@@ -78,7 +80,12 @@ ExecStart=<%= scope['kubernetes::_dest_dir'] %>/apiserver \
   --etcd-keyfile=<%= scope['kubernetes::apiserver::etcd_key_file'] %> \
 <% end -%>
   --service-cluster-ip-range=<%= scope['kubernetes::service_ip_range_network'] %>/<%= scope['kubernetes::service_ip_range_mask'] %> \
+<% if @post_1_11 -%>
+  --enable-admission-plugins=<%= @_admission_control.join(',') %> \
+  --disable-admission-plugins=<%= @_disable_admission_control.join(',') %> \
+<%- else -%>
```

Comment:
This is not really backwards compatible yet, is it?

We we build the old admission_control thing we need to filter out the default enabled ones post 1.11.

Somehow we need to expose the disable admission controller flags